### PR TITLE
Acropolis

### PR DESCRIPTION
**Pull request by bwhm**
Please review the sections relevant to you.

### CODE REVIEW

File: meetings/acropolis/README.md [hunk 1]
```diff
@@ -410,8 +411,65 @@ Before the meeting was adjourned, the issue of making the Forum work without an
...
+
+## Release Plan Milestone Evaluation Meeting
+
+**ID:** `Acropolis Release Plan Milestone Evaluation Meeting`
+- **Date:** `11.06.19`
+- **Starts:** `11:15 GMT+2`
+- **Duration:** `45 min`
+- **Venue:** `ZOOM`
+- **Lead**: `Martin`
+- **Minutes**: `Martin`
```
Comment (siman) [inline meetings/acropolis/README.md]: this looks strange, what does it mean?
`- **Minutes**: `Martin``

File: meetings/acropolis/README.md [hunk 2]
```diff
@@ -410,8 +411,65 @@ Before the meeting was adjourned, the issue of making the Forum work without an
...
+- `Bedeho`
+- `Martin`
+- `Mokhtar`
+- `Dagur` (observer)
+
+:one:
+#### Sub-System test Forum:
+- Bedeho to complete runtime build, assisted by Mokhtar/Alex
+  - To be completed by Thursday for Alex to be unblocked
+- Alex to focus on functionality, and reduce scope related to UI (e.g. not add pagination)
```
Comment (siman) [inline meetings/acropolis/README.md]: also: no substrate events.

File: testnets/acropolis/README.md [hunk 3]
```diff
@@ -57,7 +57,15 @@
 
 # Live Milestones
 
-- **TBD**
+Due to changing circumstances, a meeting [Evaluating the feasibility](../meetings/acropolis##release-plan-milestone-evaluation-meeting) of the planned [Milestones](#milestones), the following conclusion was made:
+
+| Date            |   Event                               |     Involved                            |
+| :--------------:|:-------------------------------------:|:---------------------------------------:|
+|    19.06.19     | [Spec Release](#spec-release)         | Alex, Bedeho, Mokhtar                   |
+|    17.06.19     | [Sub-system Test](#sub-system-test)   | All                                     |
```
Comment (siman) [inline testnets/acropolis/README.md]: `17.06.19` goes after `19.06.19`. is it ok?

File: meetings/acropolis/README.md (hunk 1 above)
Comment (bwhm) [reply meetings/acropolis/README.md]: `Minutes` is short for Minutes of Meeting.

**Comment by siman** [comment]
Yes, but how value for minutes can be "Martin"?

Best regards,
Alex Siman

File: meetings/acropolis/README.md (hunk 2 above)
Comment (bwhm) [reply meetings/acropolis/README.md]: Thank you, I should have included that. As @bedeho merged a little prematurely, I'll leave this unless mokhtar has any additional remkarks.

File: testnets/acropolis/README.md (hunk 3 above)
Comment (bwhm) [reply testnets/acropolis/README.md]: Yes, I decided to keep it in the same order as the "Original" Milestones.

**Comment by bwhm** [comment]
I was responsible for taking notes and submitting the minutes...

**Pull request merged**